eval.js

```diff
@@ -37,7 +39,19 @@ module.exports = function (content, filename, scope, includeGlobals) {
   var _filename = filename || module.parent.filename;
 
   if (includeGlobals) {
+    // Merge enumerable variables on `global`
     merge(sandbox, global)
+    // Merge all non-enumerable variables on `global`, including console (v10+),
+    // process (v12+), URL, etc. We first filter out anything that's already in
+    // the VM scope (i.e. those in the ES standard) so we don't have two copies
+    Object.getOwnPropertyNames(global).forEach((name) => {
+      if (!vmGlobals.includes(name)) {
+        sandbox[name] = global[name]
+      }
+    })
```

Comment:
Review hint

New globals automatically forwarded:

```js
[
  "global",
  "process",
  "Buffer",
  "atob",
  "btoa",
  "URL",
  "URLSearchParams",
  "TextEncoder",
  "TextDecoder",
  "AbortController",
  "AbortSignal",
  "EventTarget",
  "Event",
  "MessageChannel",
  "MessagePort",
  "MessageEvent",
  "clearInterval",
  "clearTimeout",
  "setInterval",
  "setTimeout",
  "queueMicrotask",
  "performance",
  "clearImmediate",
  "setImmediate",
  "__extends",
  "__assign",
  "__rest",
  "__decorate",
  "__param",
  "__metadata",
  "__awaiter",
  "__generator",
  "__exportStar",
  "__createBinding",
  "__values",
  "__read",
  "__spread",
  "__spreadArrays",
  "__spreadArray",
  "__await",
  "__asyncGenerator",
  "__asyncDelegator",
  "__asyncValues",
  "__makeTemplateObject",
  "__importStar",
  "__importDefault",
  "__classPrivateFieldGet",
  "__classPrivateFieldSet",
  "consola",
  "regeneratorRuntime",
];
```

Reply:
Most of these globals are injected by the transpiler. Only stuff like `URL` `process` `TextDecoder` etc. are of our interest.